container/kvm/initialisation.go

```diff
@@ -95,24 +89,36 @@ func getRequiredPackages(a string) []string {
 // createPool creates the libvirt storage pool directory. runCmd and chownFunc
 // are here for testing. runCmd so we can check the right shell out calls are
 // made, and chownFunc because we cannot chown unless we are root.
-func createPool(pathfinder func(string) (string, error), runCmd runFunc, chownFunc func(string) error) error {
+func ensurePool(poolInfo *libvirtPool, pathfinder func(string) (string, error), runCmd runFunc, chownFunc func(string) error) error {
 	poolDir, err := guestPath(pathfinder)
```

Comment:
We don't need this unless defining the pool, so it can go inside the if block below.

Reply:
Im not sure what you refer to...
If this refers to poolDir, it is used also in the end when setting permissions...

Reply:
Yes, I see.